### PR TITLE
Keep TOML fields alphabetized

### PR DESCRIPTION
**Pull request by cwfitzgerald**
#6366 but with config

### CODE REVIEW

File: taplo.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[formatting]
+align_comments = true
```
Comment (ErichDonGubler) [inline taplo.toml]: question: This is already the default. Is there a reason you want to specify it explicitly?

Comment (cwfitzgerald) [reply taplo.toml]: Dunno :) maybe I liked seeing both align options

File: benches/Cargo.toml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 [package]
-name = "wgpu-benchmark"
-version.workspace = true
```
Comment (teoxoy) [inline benches/Cargo.toml:3]: It's odd seeing those ordered alphabetically. I think usually people expect to see name, version, then everything else.

File: Cargo.toml [hunk 3]
```diff
@@ -1,54 +1,54 @@
 [workspace]
-resolver = "2"
-members = [
```
Comment (teoxoy) [inline Cargo.toml:3]: Same here.

Comment (ErichDonGubler) [reply Cargo.toml:3]: I think this particular case can we worked around by putting another newline in between `resolver` and the subsequent fields?

**Comment by cwfitzgerald** [comment]
This isn't landing as is

**Comment by ErichDonGubler** [comment]
@cwfitzgerald: I'm not sure on what basis you're claiming that, but from my perspective, we could work this out with consensus building. Maybe talk about this in the next maintainer's meeting? 🤔

**Comment by cwfitzgerald** [comment]
Didn't mean as anything more than "this PR needs to be rehashed before it can land, and I don't necessarily am the one to do it" 😅 - purely administrative

**Pull request closed without merging**